tests/python_tests/samples/conftest.py

```diff
@@ -0,0 +1,141 @@
+import subprocess # nosec B404
+import os
+import tempfile
+import pytest
+import shutil
+import logging
+import gc
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# Define model names and directories
+MODELS = {
+    "TinyLlama-1.1B-Chat-v1.0": { 
```

Comment:
Can we use more lightweight models with same architectures?
For example, we can replace "TinyLlama/TinyLlama-1.1B-Chat-v1.0" with "SmolLM2-136m",  "Qwen-7B-Chat" and "Qwen1.5-7B-Chat" with "Qwen2.5-0.5B-Instruct", etc. 

It should be beneficial for precommits.

Reply:
yes, we talked about that offline and will change the models. 
Moreover we are going to exclude model duplications in the sample tests.

Reply:
used smaller models, execution takes less the 10 minutes now